tests/unit/commands/deploy/test_auth_utils.py

```diff
@@ -88,6 +88,20 @@ def test_auth_per_resource_defined_on_api_resource(self):
         _auth_per_resource = auth_per_resource([Stack("", "", "", {}, self.template_dict)])
         self.assertEqual(_auth_per_resource, [("HelloWorldFunction", True)])
 
+    def test_auth_per_resource_on_non_serverless_api(self):
+        self.template_dict["Resources"]["HelloWorldApi"] = OrderedDict(
+            [
+                ("Type", "AWS::ApiGateway::RestApi"),
```

Comment:
Can we also add the test case for `AWS::ApiGatewayV2::Api`? (to make sure the branch is covered)